### PR TITLE
fix translation errors

### PR DESCRIPTION
**Pull request by wuxqing**
fix translation errors

### CODE REVIEW

**Comment by fantix** [comment]
+1 this is a good one

File: wtforms/locale/zh/LC_MESSAGES/wtforms.po [hunk 1]
```diff
@@ -164,10 +164,10 @@ msgstr "不是有效的浮点数"
 #: wtforms/fields/core.py:632
 #, fuzzy
```
Comment (fantix) [inline wtforms/locale/zh/LC_MESSAGES/wtforms.po]: Seems `fuzzy` stops the translation from taking effect

File: wtforms/locale/zh/LC_MESSAGES/wtforms.po [hunk 2]
```diff
@@ -164,10 +164,10 @@ msgstr "不是有效的浮点数"
...
 msgid "Not a valid datetime value"
-msgstr "不是有效的小数"
+msgstr "不是有效的日期与时间值"
 
 #: wtforms/fields/core.py:649
 #, fuzzy
```
Comment (fantix) [inline wtforms/locale/zh/LC_MESSAGES/wtforms.po]: ditto

**Comment by fantix** [comment]
Or run compile like this:
```bash
python setup.py compile_catalog --use-fuzzy
```

**Comment by crast** [comment]
Duplicate of #233

**Comment by fantix** [comment]
@crast referencing the wrong number? #223 is not a translation fix.

**Comment by crast** [comment]
Very sorry! Wrong PR

**Pull request merged**